mne/channels/channels.py

```diff
@@ -73,7 +73,8 @@ def _get_ch_type(inst, ch_type, allow_ref_meg=False):
     then grads, then ... to plot.
     """
     if ch_type is None:
-        allowed_types = ['mag', 'grad', 'planar1', 'planar2', 'eeg', 'csd']
+        allowed_types = ['mag', 'grad', 'planar1', 'planar2', 'eeg', 'csd',
+                         'hbo', 'hbr']
```

Comment:
Worth adding the `fnirs` subtypes while we're at it?

Reply:
good idea